sys-apps/inxi/inxi-2.3.8_p20170114.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_COMMIT=af0630e3067c138893e243896e1767c93d2a2856 #because upstream refuses to tag commits with version numbers
+MY_SHORTCOMMIT=${MY_COMMIT:0:7}
+
+DESCRIPTION="Commandline script to print hardware information for irc and administration"
+
+HOMEPAGE="https://github.com/smxi/inxi"
+SRC_URI="https://github.com/smxi/${PN}/tarball/${MY_COMMIT} -> ${P}.tar.gz"
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE=""
+
+DEPEND=""
+RDEPEND=">=app-shells/bash-3.0
+	sys-apps/pciutils
+	"
+
+S="${WORKDIR}/smxi-${PN}-${MY_SHORTCOMMIT}"
+
+src_install() {
+	dobin "${PN}"
+	doman "${PN}.1.gz"
+}
+
+pkg_postinst() {
+	einfo "To view a short or full system information:"
```

Comment:
I doubt the fitness of this into postinst. Sounds like a README/help material to me. Plus, `einfo` is only for progress messages and stuff of short usefulness, `elog` is for things user should read.

So either remove it altogether and rely on upstream documentation, or possibly use `readme.gentoo-r1` eclass.